src/core/os/amdgpu/amdgpuDevice.cpp

```diff
@@ -2050,11 +2089,25 @@ Result Device::GetSwapChainInfo(
         pSwapChainProperties->supportedUsageFlags.shaderRead    = 1;
         pSwapChainProperties->supportedUsageFlags.shaderWrite   = 1;
 
-        // Get format supported by swap chain.
-        pSwapChainProperties->imageFormatCount = static_cast<uint32>(ArrayLen(PresentableImageFormats));
+        // Get formats supported by swap chain. We have at least the 32 bpp formats.
+        pSwapChainProperties->imageFormatCount = baseFormatCount;
+
+        // Some gpu + amdgpu kms combinations do support fp16 scanout and display.
+        if (HasFP16DisplaySupport()) {
```

Comment:
ditto

Reply:
Thanks. Done, in just updated pull request.